lib/std/Target/spirv.zig

```diff
@@ -21,6 +21,7 @@ pub const Feature = enum {
     generic_pointer,
     vector16,
     shader,
+    variable_pointers,
```

Comment:
You don't have to do this now, but just FYI, `tools/update_cpu_features.zig` is now capable of generating files for targets that have no LLVM data. It would be good to move the SPIR-V stuff in there at some point so this file doesn't have to be edited by hand.